src/main/java/edu/ie3/datamodel/models/input/system/type/StorageTypeInput.java

```diff
@@ -58,15 +58,15 @@ public StorageTypeInput(
       Quantity<Power> sRated,
       double cosphiRated,
```

Comment:
Getter/Setter needs to be addressed accordingly 

```suggestion
      double cosPhiRated,
```